### ISSUE TITLE
runASR - Input sequence file in fasta format throws error 

### ISSUE THREAD

**Issue by julianzaugg** [bodenlab/ASR#2]
Calling runASR using a sequence file in FASTA format throws error

```R
treeLoc <- "UPGMA_cyp102A_tree.txt"
alignmentLoc <- "CYP102A_chim_seqs_reduced_fasta.txt"

runASR(treeLoc, alignmentLoc)

    Exception in thread "main" java.lang.RuntimeException: Alignment should be in Clustal or Fasta format
	at bn.reconstruction.ASR.loadData(ASR.java:114)
	at bn.reconstruction.ASR.<init>(ASR.java:52)
	at bn.reconstruction.runASR.main(runASR.java:27)
    Error in file(file, "rt") : cannot open the connection
    In addition: Warning message:
    In file(file, "rt") :
      cannot open file 'temp_new_tree.txt': No such file or directory
```
[CYP102A_chim_seqs_reduced_fasta.txt](https://github.com/bodenlab/ASR/files/56393/CYP102A_chim_seqs_reduced_fasta.txt)
[UPGMA_cyp102A_tree.txt](https://github.com/bodenlab/ASR/files/56394/UPGMA_cyp102A_tree.txt)

**Comment by aesse** [comment]
The check for Clustal or Fasta alignment has been fixed in bnkit - see commit 9a471b6

**Comment by aesse** [comment]
ASR.jar in the R package has been updated with the changes to bnkit
